shiksha-website/shiksha-frontend/src/app/shared/components/baseline-survey/baseline-survey.component.scss

```
@@ -1,618 +1,569 @@
+  --line: #E2E8F0;
+  --panel: #F8FAFC;
+  --success: #2E9E6F;
+  --danger: #DC2626;
+  --warn-bg: #FFF3E0;
+  --warn-border: #FFE0B2;
+  --warn-text: #92601A;
+  --disabled-bg: #F1F5F9;
+  --disabled-border: #E2E8F0;
+  --disabled-text: #94A3B8;
+
+  font-family: 'Inter',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)